deploy/clowdapp.yaml

```diff
@@ -137,8 +137,8 @@ objects:
 parameters:
 - name: INGRESS_STAGEBUCKET
   value: insights-upload-perma
-- name: INGRESS_VALIDTOPICS
-  value: advisor,compliance,hccm,qpc,rhv,tower,leapp-reporting,buckit,xavier,mkt,playbook,playbook-sat,resource-optimization,malware-detection,pinakes,assisted-installer,runtimes-java-general
+- name: INGRESS_VALID_UPLOAD_TYPES
+  value: advisor,compliance,hccm,qpc,rhv,tower,leapp-reporting,buckit,xavier,mkt,playbook,playbook-sat,resource-optimization,malware-detection,pinakes,assisted-installer,runtimes-java-general,openshift
```

Comment:
Do we still need `buckit` here with the addition of `openshift`?  Not sure sure, so just asking.

Reply:
I think we can safely remove `buckit` because the code was mapping `openshift` upload type to the `buckit` topic.